source/style-guide/writing/use-effective-verbs.txt

```diff
@@ -329,9 +329,7 @@ Be careful not to ascribe human feelings, motivations, and actions to
 inanimate objects. For example, a web application doesn't *know*,
 *need*, *remember*, *see*, *think*, *understand*, or *want*.
 
-Be careful not to ascribe human qualities, traits, or emotions to
-inanimate objects like software, hardware, or interfaces. Often, we use
-anthropomorphic words in an attempt to make difficult topics more understandable or relatable. This can actually make the
+Often, we use anthropomorphic words in an attempt to make difficult topics more understandable or relatable. This can actually make the
```

Comment:
**suggestion:** As written, this is hard to follow. I would recommend condensing.

```suggestion
Often, we use words that anthropomorphize (or humanize) to simplify topics. This can actually make the
```

Reply:
Reworded.